tests/smb/smb_load.py

```diff
@@ -318,15 +325,20 @@ def delete(smbclient, load_test_config, process):
         raise OperationFailedError(f"delete operation failed, Error {e}")
 
 
-def cleanup(client, load_test_config):
+def cleanup(
```

Comment:
One question here Mohit, will this impact the existing cases? Are we using this somewhere else also? If its specific to this scenario then please ignore, rest looks good to me

Reply:
Yes @pranavprakash20 
It is specific to this test only.